pkg/controller/volume/cache/actual_state_of_world.go

```diff
@@ -47,7 +55,7 @@ type ActualStateOfWorld interface {
 	// added.
 	// If no node with the name nodeName exists in list of attached nodes for
 	// the specified volume, the node is added.
-	AddVolumeNode(volumeSpec *volume.Spec, nodeName string) (api.UniqueDeviceName, error)
+	AddVolumeNode(volumeSpec *volume.Spec, nodeName string) (api.UniqueVolumeName, error)
```

Comment:
Looking at this more closely, it's technically not adding Volume *to* node but vice versa. And that detail is irrelevant to the caller. The caller is simply interested in adding the given volume/node combo to the store, that's it. So I could be convinced to rename it to AddVolumeNodeCombo, but I'd prefer to leave it as is. Let me know if you feel strongly.